integration/test/Test/Brig.hs

```diff
@@ -1,11 +1,12 @@
+{-# OPTIONS_GHC -Wno-incomplete-uni-patterns #-}
```

Comment:
i'm also ok with doing this in integration.cabal...

Reply:
Create a PR then.